packages/quantic/cypress/integration/sort/sort-expectations.ts

```diff
@@ -0,0 +1,48 @@
+import {InterceptAliases} from '../../page-objects/search';
+import {SearchExpectations} from '../search-expectations';
+import {SortSelector, SortSelectors} from './sort-selectors';
+
+function sortExpectations(selector: SortSelector) {
+  return {
+    selectedOption: (value: string, selected: boolean) => {
+      selector.selectedOption(value).should(selected ? 'exist' : 'not.exist');
+    },
+
+    displaySortDropdown: (display: boolean) => {
+      selector.listbox().should(display ? 'exist' : 'not.exist');
+    },
+
+    containsOptions: (values: string[]) => {
+      values.forEach((value) => {
+        selector.option(value).should('exist');
+      });
+    },
+
+    displayLocalizedLabel: (label: string) => {
+      selector.label().should('contain', label);
+    },
+
+    displayLocalizedOptionLabels: (values: string[], labels: string[]) => {
```

Comment:
Why not pass a single object that contains the value/label mappings? Or maybe an array of option? With 2 distinct arguments, we can end up with mismatches between `values` and `labels`.

I would suggest to rename the expectation `optionsEqual` instead.